channels/pkg/channels/channel_version.go

```diff
@@ -209,3 +209,25 @@ func (c *Channel) SetInstalledVersion(ctx context.Context, k8sClient kubernetes.
 	}
 	return nil
 }
+
+// Remove removes the channel from the namespace.
```

Comment:
Nit: "Removes the channel tracking-annotation from the namespace" might be less scary-sounding!